yarn.lock

```diff
@@ -0,0 +1,11716 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
@anass412  let's add this to gitignore & remove it

Reply:
i will :)